portfolio/src/main/java/com/google/sps/data/Comment.java

```diff
@@ -21,12 +21,13 @@
 @AutoValue
 public abstract class Comment {
 
-  public static Comment create(long id, Date date, String text, String author) {
-    return new AutoValue_Comment(id, date, text, author);
+  public static Comment create(long id, Date date, String text, String author, String email) {
+    return new AutoValue_Comment(id, date, text, author, email);
   }
 
   public abstract long getId();
```

Comment:
Could you add documentations for those fields? It's not very clear to me what "Id" is. Is it the comment ID or the author ID?

Reply:
Done!